src/components/home/styledHome.js

```diff
@@ -0,0 +1,50 @@
+import styled from "styled-components";
+
+export const HomeSection = styled.section`
+    display: flex;
+    flex-direction: column;
+    width: 94%;
+    padding: 74px 0 40px 0;
+    margin: auto;
+`
+
+export const Title = styled.h1`
+    width: 59.65vw;
+    margin-bottom: 85px;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 44px;
+    line-height: 48px;
+    color: #1C383C;
+`
+export const TitleSpan = styled.span`
+    color: #FF9820;
+`
+export const HomeGraphics = styled.div`
+    display: flex;
+    justify-content: space-between;
+`
+export const HomePicContainer = styled.div`
+    width: 43.6vw;
```

Comment:
Same as the earlier comment. I'd recommend having a max-width or something to cap how large the container can grow.

Height of the image should be 369